pkgs/applications/misc/houdini/runtime.nix

```diff
@@ -50,11 +58,13 @@ stdenv.mkDerivation rec {
                       --no-root-check \
                       --accept-EULA \
                       $out
-    sed -i "s|/usr/lib/sesi|${license_dir}|g" $out/houdini/Licensing.opt
+    echo -e "localValidatorDir = ${license_dir}\nlicensingMode = localValidator" > $out/houdini/Licensing.opt
```

Comment:
Is changing licensing scheme the reason they broke checksums? I am not sure how insightful a request towards support@sidefx.com would be to ask, if they plan to continue this practice in future.